### ISSUE TITLE
Exporting live data should snapshot it first so uuid doesn't get confused when reimporting.

### ISSUE THREAD

**Issue by cmeyer** [nion-software/nionswift#153]
See also
- #592 
- https://github.com/nion-software/nionswift/issues/430#issuecomment-1133518706

**Comment by Brow71189** [comment]
Maybe one could also do it the other way round: Assign a new uuid to important items. The reason I'm proposing this is that it would also help avoiding uuid confusion when importing a data item twice.

**Comment by cmeyer** [comment]
Do you mean while importing? Snapshot already assigns new uuid.

**Comment by Brow71189** [comment]
Yes I mean you would assign a new uuid during import. I'm not sure if this creates other issues. But the uuid confusion when you import the same data multiple times is also really annoying.
I had this specific issue when working with files exported in raw numpy format. I wanted to keep the calibrations, so I just edited the .npy file and imported it again (and did that multiple times). Of course all of these data items ended up having the same uuid.

**Comment by cmeyer** [comment]
Note: see how this is handled in nhdf.
